requirements.txt

```diff
@@ -126,23 +126,24 @@ six==1.16.0
     #   furl
     #   orderedmultidict
     #   python-dateutil
-sqlalchemy==2.0.25
+sqlalchemy==1.4.22
     # via
+    #   -r pip-tools/requirements.in
     #   alembic
     #   flask-sqlalchemy
     #   sqlalchemy-utils
 sqlalchemy-utils==0.41.1
     # via -r pip-tools/requirements.in
 typing-extensions==4.9.0
-    # via
-    #   alembic
-    #   sqlalchemy
+    # via alembic
 urllib3==1.26.18
     # via
     #   botocore
     #   requests
-werkzeug==3.0.1
-    # via flask
+werkzeug==2.0.3
```

Comment:
Not sure if this impacts this repo, but the changes made to cf-hello-worlds modified two packages. one of which is Werkzeug 2.2.2:
https://github.com/cloud-gov/cf-hello-worlds/pull/127/files

Reply:
Ah good to know.